roster/models.py

```diff
@@ -112,6 +112,11 @@ class Student(models.Model):
 	enabled = models.BooleanField(
 		default=True, help_text="Allow student to submit/request units."
 	)
+	public = models.BooleanField(
```

Comment:
I would have named this something longer since it's not clear to an outsider what `public` should mean on a student object (maybe, `show_on_leaderboard`?). Might be a pain to change now though, so I can also just do it myself later.